pkg/splunk/enterprise/util.go

```diff
@@ -1395,6 +1392,19 @@ func shouldCheckAppRepoStatus(client splcommon.ControllerClient, cr splcommon.Me
 	return false
 }
 
+// getCleanObjectDigest returns only hexa-decimal portion of a string
+// Ex. '\"b38a8f911e2b43982b71a979fe1d3c3f\"' is converted to b38a8f911e2b43982b71a979fe1d3c3f
+func getCleanObjectDigest(dirtyObjectDigest *string) (*string, error) {
```

Comment:
nit: maybe a better name would be `extractObjectHash`?

Reply:
Here in this case it is really dirty, with those unnecessary `""\` characters. if it sounds aggressive, probably, we can make it as rawObjectHash :-)